services/ui/serverless.yml

```diff
@@ -88,6 +88,32 @@ resources:
               Principal:
                 CanonicalUser: !GetAtt CloudFrontOriginAccessIdentity.S3CanonicalUserId
         Bucket: !Ref S3Bucket
+    LoggingBucket:
+      Type: "AWS::S3::Bucket"
+      Properties:
+        BucketName: !Sub ${self:service}-${self:custom.stage}-cloudfront-logs
```

Comment:
we should append the account ID to the end of this bucket as the bucket is a global name space and as folks fork this repo or pull this change in to address their own products they'd very likely have a naming collision
https://github.com/CMSgov/macpro-quickstart-serverless/blob/master/services/uploads/serverless.yml#L100
See above as an example

Reply:
good catch, thanks

Reply:
Requested change has been applied. Thanks!